todo_app/data/trello_items.py

```diff
@@ -10,8 +9,12 @@
 
 
 class Item:
+    TODO = "To Do"
+    DONE = "Done"
     ''' Class to represent an item (card)'''
-    def __init__(self, id, name, status='To Do'):
+    def __init__(self, id, name, status=None):
+        if not status:
+            status = Item.TODO
```

Comment:
Again minor, but you could just specify the default directly:
```suggestion
    def __init__(self, id, name, status=Item.TODO):
```

I guess this doesn't catch the case that a user intentionally puts in a `None` status, but I think that's reasonable

Reply:
This wouldn't work - you don't have Item or even self in the declaration of init